Mod.cs

```diff
@@ -26,23 +26,26 @@ public class Mod : StardewModdingAPI.Mod
 
         public override void Entry(IModHelper helper)
         {
+            Monitor.Log($"GMCM version {ModManifest.Version} loading...", LogLevel.Info);
             instance = this;
             Log.Monitor = Monitor;
+            Api.apiLog = Monitor.Log;
 
             ui = new RootElement();
 
             Texture2D tex = Helper.Content.Load<Texture2D>("assets/config-button.png");
-            configButton = new Button(tex);
-            configButton.LocalPosition = new Vector2(36, Game1.viewport.Height - 100);
-            configButton.Callback = (Element e) =>
-            {
-                Game1.playSound("newArtifact");
-                TitleMenu.subMenu = new ModConfigMenu( false );
-            };
+            configButton = new Button(tex) {
+                LocalPosition = new Vector2(36, Game1.viewport.Height - 100),
+                Callback =
+                    (Element e) => {
+                        Game1.playSound("newArtifact");
+                        TitleMenu.subMenu = new ModConfigMenu(false);
+                    }
+                };
```

Comment:
Originally changed this to `uiViewport` as well, but apparently Title Screen uses `viewport` instead.

So, I reverted my change, but kept the "new-style" object property initialization syntax.